node/standalone/Cargo.toml

```diff
@@ -58,8 +59,13 @@ sp-block-builder = { git = "https://github.com/paritytech/substrate", branch = "
 
 moonbeam-runtime = {path = "../../runtime", default-features = false, features = ["std", "standalone"] }
 moonbeam-rpc = { path = "../rpc" }
-fc-consensus = { git = "https://github.com/purestake/frontier", branch = "v0.4-hotfixes" }
+fc-consensus = { git = "https://github.com/purestake/frontier", branch = "v0.5-hotfixes" }
+fp-consensus = { git = "https://github.com/purestake/frontier", branch = "v0.5-hotfixes" }
 
+ethereum = { package = "pallet-ethereum", git = "https://github.com/purestake/frontier", branch = "v0.5-hotfixes" }
+frontier-rpc = { package = "fc-rpc", git = "https://github.com/purestake/frontier", branch = "v0.5-hotfixes" }
+frontier-rpc-primitives = { package = "fp-rpc", git = "https://github.com/purestake/frontier", branch = "v0.5-hotfixes" }
+fc-rpc-core = { git = "https://github.com/purestake/frontier", branch = "v0.5-hotfixes" }
```

Comment:
More aliases. I guess this #163 didn't merge in that nicely?

Reply:
My bad. It produced a lot of conflicts in the `Cargo.toml` files. I was focused on the `frontier-` aliases; I should have focused on `package =` instead.